farm-staking/farm-staking-proxy/src/external_contracts_interactions.rs

```diff
@@ -59,7 +59,7 @@ pub trait ExternalContractsInteractionsModule:
         let lp_farm_address = self.lp_farm_address().get();
         let exit_farm_result: ExitFarmResultType<Self::Api> = self
             .lp_farm_proxy_obj(lp_farm_address)
-            .exit_farm_endpoint(orig_caller)
+            .exit_farm_endpoint(lp_farm_token_amount.clone(), orig_caller)
```

Comment:
you need an input here as well - as in case of Proxy_EGLD_LKMEX token I still want to be able to partially exit without the need of 3 transactions

Reply:
Updated.